docs/src/main/paradox/caffeine.md

```diff
@@ -0,0 +1,52 @@
+# Caffeine Flow
+
+The [Caffeine](https://github.com/ben-manes/caffeine) flow provides Akka Stream flow, to reconcile data in memory.
+
+
+Let A(1), A(2), A(3), ...., B(1), C(1) event to enter the flow, this flow will emit (A(1), B(1), C(1)) on reconciled OutLet and (A(2), A(3)) on Expired OutLet
```

Comment:
I would not understand when I should use this component based on above description. I'm not familiar with Caffeine.